.circleci/config.yml

```diff
@@ -126,7 +126,7 @@ jobs:
             fi
 
             export CARGO_INCREMENTAL=0
-            "${cov_prefix[@]}" cargo build $CARGO_FLAGS --bins --tests
+            "${cov_prefix[@]}" cargo build $CARGO_FLAGS --features fail/failpoints --bins --tests
```

Comment:
Do we need a similar change for `cargo build` in Makefile?

Reply:
I do not want failpoints to be linked in production. It should be used only for tests. When we are doing "cargo build" there is no information whether we are doing it to run tests or we are building images for production. Certainly we can create special make target for building zenith with safepoints. For me  manual specification of `--features fail/failpoints` works fine. Is there any other way except creating special make target for  builds with safepoints?

Including safepoints only in debug builds IMHO is not also good option, because debug version is soooo slow.